dxr/plugins/clang/condense.py

```diff
@@ -17,6 +17,10 @@
 from dxr.indexers import FuncSig, Position, Extent
```

Comment:
I notice that both files have had their permissions changed (see "100644 → 100755") on the "Files changed" tab. I'm sure you didn't mean to do that, so have a look at your setup and see what might be changing permissions behind your back. If Windows is involved, that's one place I'd look.